### ISSUE TITLE
Results are similar to the original deeplab v2 paper?

### ISSUE THREAD

**Issue by dyz-zju** [zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow#5]
Hello, your implementation has achive the results compared to the original paper?
I want to known,thanks!

**Comment by zhengyang-wang** [comment]
Compared to the original deeplabv2 paper, two things were not implemented:
1. MSC: Employing mutli-scale inputs with max fusion.
2. CRF

My mIoU on val is 75.1, which is reasonable.

**Comment by dyz-zju** [comment]
Hello, I just run your code on augmented pascalvoc datasets, and not change anything. 
After iter 20000, I test just get mIoU 68.8%, do you know why? Thanks!

**Comment by dyz-zju** [comment]
And more, the loss just shocks, after iter 20000, the final loss is aroud 1.3.

**Comment by zhengyang-wang** [comment]
Which pre-trained model did you use?

**Comment by zhengyang-wang** [comment]
And you may notice that you need to change 'valid_step' in main.py for testing/validation.

**Comment by dyz-zju** [comment]
I use deeplab_resnet_init.ckpt as my pertained model，and valid step is changed to 20000.

**Comment by zhengyang-wang** [comment]
Your final loss is correct. I cannot figure out what happened in your case without more information. I tested the code for several runs, the mIoU results for val are consistently around 75.

**Comment by licaizi** [comment]
@dyz-zju 您好，我也只跑出了0.68，请问后来您的结果怎么样了

**Comment by zhengyang-wang** [comment]
Please check https://github.com/zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow/issues/7

**Comment by dyz-zju** [comment]
@CaiziLee you need to use another dataset processed by author, the url can be found in somewhere.
